### DISCUSSION TITLE
Custom toolbar with Editor component

### DISCUSSION THREAD

**Discussion by MaximeRobion** [primefaces/community#1423] [PrimeNG]
Hello,
Using prime v13, and the editor component https://www.primefaces.org/primeng-v13-lts/#/editor (with Quill under the hood). I don't want image as an option in the toolbar so I'm creating a custom one.
Though, header and list (from Quill formats: https://quilljs.com/docs/formats) don't work as the rest: bold, italic, strike...

I would need guidance on how to add these header and list to my toolbar.
My current code:

```
<p-editor
    [(ngModel)]="value"
    [style]="{ height: height + 'px' }"
    [placeholder]="placeholder"
    [readonly]="readonly"
    (onTextChange)="onTextChange($event)">
    <ng-template pTemplate="header">
        <span class="ql-formats">
            <button type="button" class="ql-bold" aria-label="Bold"></button>
            <button type="button" class="ql-italic" aria-label="Italic"></button>
            <button type="button" class="ql-underline" aria-label="Underline"></button>
            <button type="button" class="ql-strike" aria-label="Strike"></button>
            <button type="button" class="ql-header" aria-label="Header"></button>
            <button type="button" class="ql-list" aria-label="List"></button>
        </span>
    </ng-template>
</p-editor>

```

PS: The documentation is weird. "Toolbar" code doesn't work, but the source code for "Custom Toolbar" does.

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue?

**Comment by cetincakiroglu** [comment]
Closing due to the inactivity.
